go/felix/proto/felixbackend.proto

```diff
@@ -145,8 +145,9 @@ message PolicyID {
 }
 
 message Policy {
-  repeated Rule inbound_rules = 1;
-  repeated Rule outbound_rules = 2;
+  bool untracked = 1;
```

Comment:
Adding the untracked flag here ads it to the API between the go and python parts but I don't see any changes on the golang side of the API to actually fill in the flag with the correct value.  I think you need to set the flag around event_buffer.go:209:
```
		Policy: &proto.Policy{
			InboundRules:  parsedRulesToProtoRules(rules.InboundRules),
			OutboundRules: parsedRulesToProtoRules(rules.OutboundRules),
		},
```
You'll also need to add the flag to the datamodel in libcalico-go so that it's available here.  Felix uses the backend datamodel, so you'll need to add it to the Policy struct in `lib/backend/model/policy.go`:

```
type Policy struct {
	Order         *float64 `json:"order,omitempty" validate:"omitempty"`
	InboundRules  []Rule   `json:"inbound_rules,omitempty" validate:"omitempty,dive"`
	OutboundRules []Rule   `json:"outbound_rules,omitempty" validate:"omitempty,dive"`
	Selector      string   `json:"selector" validate:"selector"`
}
```